### ISSUE TITLE
How is the performance by the default RandLANet on validation set? 

### ISSUE THREAD

**Issue by yzheng97** [QingyongHu/SensatUrban#14]
I got maximum mIoU of 45.6 at epoch 80.

**Comment by Lionel2021-belgique** [comment]
Hi; 
could you please share with me the pre-trained model that you have obtained
Best regards

**Comment by codename1995** [comment]
> I got maximum mIoU of 45.6 at epoch 80.

I got a max mIoU of 48.02, which is still far from the reported result in the paper (58.64 in Table 3).
